pkg/cliplugins/workload/plugin/sync.go

```diff
@@ -673,8 +676,8 @@ type templateInput struct {
 	KCPNamespace string
 	// Namespace is the name of the syncer namespace on the pcluster
 	Namespace string
-	// LogicalCluster is the qualified kcp logical cluster name the syncer will sync from
-	LogicalCluster string
+	// SyncTargetPath is the qualified kcp logical cluster name the syncer will sync from
+	SyncTargetPath string
```

Comment:
If this is a name and not a path, let's rename to SyncTargetName?

Reply:
Or SyncTargetClusterName?

Reply:
path. We translate to cluster name internally.